CONTRIBUTING.md

```diff
@@ -203,6 +203,45 @@ $ docker run --rm -it -v /path/where/you/have/cloned/the/repository\:/src:ro -v
 This configuration should generally cover  your needs. For more details about other configurations, see [documentation in the SyTest repo](https://github.com/matrix-org/sytest/blob/develop/docker/README.md).
 
 
+## Run the integration tests ([Complement](https://github.com/matrix-org/complement))
```

Comment:
There is also some Complement Docker image building docs at [`docker/README-testing.md`](https://github.com/matrix-org/synapse/blob/b7186c6e8ddacc328ae2c155162b36291a3c2b79/docker/README-testing.md) but it doesn't seem to target a Synapse dev just trying to write a feature and some tests like this PR is trying to do.